src/main/java/seedu/cafectrl/data/Menu.java

```diff
@@ -39,8 +39,8 @@ public Dish getDishFromName(String dishName) {
         String formattedDishName = dishName.toLowerCase().trim();
         for (int i = 0; i < getSize(); i++) {
             String menuDishName = getDishFromId(i).getName();
-            String formattedMenuDishName = menuDishName.toLowerCase().trim();
-            if (formattedMenuDishName.equals(formattedDishName)){
+            String formattedMenuDishName = menuDishName.trim();
+            if (formattedMenuDishName.equalsIgnoreCase(formattedDishName)){
```

Comment:
perhaps add whitespace
```suggestion
            if (formattedMenuDishName.equalsIgnoreCase(formattedDishName)) {
```